src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/extension.ts

```diff
@@ -177,6 +179,13 @@ export async function activate(vscodeType: typeof vscodeapi, context: ExtensionC
                 vscodeType.languages.registerDocumentHighlightProvider(
                     RazorLanguage.id,
                     documentHighlightProvider),
+                // Our OnTypeFormatter doesn't do anything at the moment, but it's needed so
+                // VS Code doesn't throw an exception when it tries to send us an
+                // OnTypeFormatting request.
+                vscodeType.languages.registerOnTypeFormattingEditProvider(
```

Comment:
Should resolve https://github.com/dotnet/razor/issues/8175

Reply:
Why dont we support on type formatting?

Reply:
I just don't think we ever did in VS Code? We should support it eventually and I don't think (crossing fingers) it should be too difficult. However, it would definitely require a decent amount of validation so I left it out of this PR for now. I filed a tracking issue here: https://github.com/dotnet/razor/issues/8329